src/components/ContentPreview/ContentPreview.js

```diff
@@ -69,7 +69,7 @@ type Props = {
     sharedLink?: string,
     sharedLinkPassword?: string,
     onError?: Function,
-    onMetric?: Function,
+    onMetric: Function,
```

Comment:
wrappers/ContentPreview.js doesn't pass this btw

Reply:
Yeah, it is added in ```defaultProp``` which defaults it to noop.